frontera/contrib/scrapy/schedulers/frontier.py

```diff
@@ -124,7 +128,8 @@ def process_exception(self, request, exception, spider):
     def open(self, spider):
         log.msg('Starting frontier', log.INFO)
         if not self.frontier.manager.auto_start:
-            self.frontier.start()
+            settings_override = getattr(spider, 'frontier_settings', {})
```

Comment:
I think allowing to change Frontera manager settings on the fly creates a surface for potential hardly debuggable problems. What about instantiating ``self.frontier`` in ``open()`` with already adjusted settings?